### ISSUE TITLE
Invalid card numbers get logged to the console vertically 

### ISSUE THREAD

**Issue by Leprekus** [Leprekus/Credit-Card-Checker#1]
Writing: 
`console.log(findInvalidCards(batch))`

Outputs:
```
[
  'INVALID CARDS:',
  [ 4, 5, 3, 2, 7, 7, 8, 7, 7, 1, 0, 9, 1, 7, 9, 5, [ 'Visa', 4 ] ],
  [
    5,
    7,
    9,
    5,
    5,
    9,
    3,
    3,
    9,
    2,
    1,
    3,
    4,
    6,
    4,
    3,
    [ 'Mastercard', 5 ]
  ],
  [
    3,
    7,
    5,
    7,
    9,
    6,
    0,
    8,
    4,
    4,
    5,
    9,
    9,
    1,
    4,
    [ 'Amex (American Express)', 3 ]
  ],
  [ 6, 0, 1, 1, 1, 2, 7, 9, 6, 1, 7, 7, 7, 9, 3, 5, [ 'Discover', 6 ] ],
  [
    5,
    3,
    8,
    2,
    0,
    1,
    9,
    7,
    7,
    2,
    8,
    8,
    3,
    8,
    5,
    4,
    [ 'Mastercard', 5 ]
  ],
  [
    3,
    4,
    4,
    8,
    0,
    1,
    9,
    6,
    8,
    3,
    0,
    5,
    4,
    1,
    4,
    [ 'Amex (American Express)', 3 ]
  ],
  [
    6,
    0,
    1,
    1,
    3,
    7,
    7,
    0,
    2,
    0,
    9,
    6,
    2,
    6,
    5,
    6,
    2,
    0,
    3,
    [ 'Discover', 6 ]
  ],
  [ 4, 9, 2, 9, 8, 7, 7, 1, 6, 9, 2, 1, 7, 0, 9, 3, [ 'Visa', 4 ] ]
]
```

**Comment by LancelotHayward** [comment]
I got an error
`> console.log(findInvalidCards(batch))
Uncaught TypeError: func is not a function
    at REPL93:12:34
    at Array.forEach (<anonymous>)
    at findInvalidCards (REPL93:12:15)`
 But replacing func(card) with validateCred(card) solved that for me.
 Afterwards I tested making visa longer (visalololololololol) and it made visa also log vertically.
 It seems node.js does this when there is a sufficient amount of characters in arrays. (I've never used node.js as I'm new to all of this)
 I wrote a line which is a botched solution for your issue. It's in a pull request
